docs/src/content/docs/fr/guides/pages.mdx

```diff
@@ -0,0 +1,151 @@
+---
+title: Pages
+description: Apprenez à créer et à gérer les pages de votre site de documentation avec Starlight.
+sidebar:
+  order: 1
+---
+
+Starlight génère les pages HTML de votre site en fonction de votre contenu, avec des options flexibles fournies par le biais du frontmatter en Markdown.
+En outre, les projets Starlight bénéficient d'un accès complet aux [puissants outils de génération de pages d'Astro](https://docs.astro.build/fr/basics/astro-pages/).
+Ce guide montre comment fonctionne la génération de pages dans Starlight.
+
+## Pages de contenu
+
+### Formats de fichiers
+
+Starlight prend en charge la création de contenu en Markdown et MDX sans aucune configuration requise.
+Vous pouvez ajouter la prise en charge de Markdoc en installant l'intégration expérimentale [Astro Markdoc](https://docs.astro.build/fr/guides/integrations-guide/markdoc/).
+
+### Ajouter des pages
+
+Ajoutez de nouvelles pages à votre site en créant des fichiers `.md` ou `.mdx` dans `src/content/docs/`.
+Utilisez des sous-dossiers pour organiser vos fichiers et pour créer plusieurs segments de chemin.
+
+Par exemple, la structure de fichier suivante générera des pages à `exemple.com/hello-world` et `exemple.com/reference/faq` :
+
+import FileTree from '~/components/file-tree.astro';
+
+<FileTree>
+
+- src/
+  - content/
+    - docs/
+      - hello-world.md
+      - reference/
+        - faq.md
+
+</FileTree>
+
+### Sécurité de la typologie du frontmatter
```

Comment:
```suggestion
### Frontmatter avec sûreté du typage
```

We usually use "typage" ([reference](https://www.larousse.fr/dictionnaires/francais/typage/186838)) as "typologie" refers to an entirely different domain.

Reply:
Oh great!